CSS/styles.css

```diff
@@ -77,17 +78,48 @@ ol{
 }
 
 .rules-container{
-    border : 3px solid yellow;
+    /* border : 3px solid yellow; */
     display : flex;
     flex-direction: column;
     gap : 3rem;
     width : 60%;
 }
 
+.categories-flex-container{
+    /* border : 3px solid yellow; */
+    display : flex;
+    flex-direction: row;
+    justify-content : center;
+    gap : 2rem;
+}
 
+.single-category-container{
+    /* border : 3px solid green; */
+    display : flex;
+    flex-direction: column;
+    padding : 1.2rem;
+    width : 35rem;
+    gap : 10px;
+    border-radius: 0.8rem;
+    background-color : var(--light-test-color);
+    cursor : pointer;
+    /* height : 25rem; */
+}
 
+.img-container{
+    /* border : 3px solid purple; */
+    width : 100%;
+    height : 20rem;
```

Comment:
The files have way more than redundant comments. If you are not planning to use these in any way you should remove these. Other than that everything is great. Great work.